app/controllers/clients_controller.rb

```diff
@@ -69,15 +72,23 @@ def update
   # guests form sends xhr request to this endpoint
   # new clients are normally created in conjunction with intakes in intakes#create
   def create
-    client_params = params.require("client").permit(:first_name, :last_name, :date_of_birth, :profile_photo, :camera)
-    if client_params["date_of_birth"].blank?
-      client_params["date_of_birth"] = "1600-01-01"
+    client_params =
+      params
+        .require('client')
+        .permit(
+          :first_name,
+          :last_name,
+          :date_of_birth,
+          :hmis_id,
```

Comment:
I think this action only comes from guests (which doesnt record hmis_id), but adding to whitelist here just to be safe.